public/app/plugins/panel/geomap/module.tsx

```diff
@@ -121,5 +121,17 @@ export const plugin = new PanelPlugin<GeomapPanelOptions>(GeomapPanel)
         name: 'Show debug',
         description: 'Show map info',
         defaultValue: false,
+      })
+      .addRadio({
+        category,
+        path: 'tooltip.mode',
+        name: 'Tooltip',
+        defaultValue: TooltipMode.Details,
+        settings: {
+          options: [
+            { label: 'None', value: TooltipMode.None, description: 'Show contents on click, not hover' },
+            { label: 'Details', value: TooltipMode.Details, description: 'Show popup on hover' },
```

Comment:
Might be a little clearer to end user maybe if name was `Tooltip Display` and options were `Hover` vs `Click`? Though that may be a little confusing given user can "click" in hover mode to persist tooltip so 🤷